index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
           <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon" class="space"></a></li>
           <li><a href="#"><img src="image/Vector-(2).png" alt="Social media" class="space"></a></li>
         </ul>
-      </section>
-  </div>
-</main>
+     </section>
+    </div>
+  
+    <div class="cards">
+      <div class="card">
+        <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page">
+        <div class="card-content">
+          <div class="card-heading">Tonic</div>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <a href="#" class="detail-style2">See Project</a>
+        </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(1).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
```

Comment:
- [ ] As mentioned above, please use semantic tag to wrap your element, a div can not have as a child  an HTML string not warped, please fix this too